### ISSUE TITLE
Security

### ISSUE THREAD

**Issue by jaackkiscool** [vesperlol/Monkey-Obfuscator#2]
Hey, the security is **not secure.** But good work for contributing this to newbies!

**Comment by 0sir1ss** [comment]
Yes, obfuscators that use the exec function don't provide any real security, it just stops people from initially seeing the source code.
However replace exec with print and you get the source code so

**Comment by zvxfdev** [comment]
> Yes, obfuscators that use the exec function don't provide any real security, it just stops people from initially seeing the source code.
> However replace exec with print and you get the source code so

Actually you get compiled pyc code. You can use other tools like uncompyl6 to get your code from that

@0sir1ss

**Comment by vesperlol** [comment]
> > Yes, obfuscators that use the exec function don't provide any real security, it just stops people from initially seeing the source code.
> > However replace exec with print and you get the source code so
> 
> Actually you get compiled pyc code. You can use other tools like uncompyl6 to get your python code from that
> 
> @0sir1ss

and whats cool its that uncompyle6 and decompyle3 doesnt work on 3.9, so use Python 3.9 to obfuscate your code ;)

**Comment by zvxfdev** [comment]
> > 
> 
> its working on every py version lol u just have to edit a bit

what is @6nz

**Comment by 6nz** [comment]
> > > 
> > 
> > its working on every py version lol u just have to edit a bit
> 
> what is @6nz 

?
